package.json

```diff
@@ -26,4 +26,8 @@
       "eslint  --fix"
     ]
   }
+  "dependencies": {
+    "@nestjs/common": "^9.2.1",
```

Comment:
Not sure if it matters, but should we remove the carat to use the specific version here?